Utils/gen-unicode-data/Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version:5.4
+
+import PackageDescription
+
+let package = Package(
+  name: "GenUnicodeData",
+  platforms: [.macOS(.v10_15)],
+  targets: [
+    .target(
+      name: "GenUtils",
+      dependencies: []
+    ),
+    .executableTarget(
+      name: "GenGraphemeBreakProperty",
+      dependencies: ["GenUtils"]
+    ),
+    .executableTarget(
+      name: "GenNormalization",
+      dependencies: ["GenUtils"]
+    ),
+    .executableTarget(
+      name: "GenScalarProps",
+      dependencies: ["GenUtils"]
+    )
+  ]
+)
```

Comment:
Why is this a nested package?

Reply:
It can be part of the string processing package if you feel it should be, but it's a pretty stand alone package that is very severable.